### PR TITLE
Docs: Move content from advanced to getting started.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Move content from advanced to getting started.
Closes https://github.com/cksource/ckeditor5-internal/issues/2373.

---

### Additional information

Changes in this PR affect other PR: https://github.com/cksource/collaboration-features/pull/4922. The other PR should be merged just after this one.

### CODE REVIEW

**Review by apadol-cksource — APPROVED** [review]
The change looks more than good to me. The outcome is exactly as expected. We could potentially change the order of the articles in TOC as I have a feeling that getting and setting data is used more often than listening to changes in the editor but we can easily change it later. Let's double check it with extra set of eyes as there is plenty of changes in many places. Maybe somebody from QA?

**Review by dufipl — CHANGES_REQUESTED** [review]
Suddenly, demos are not working here: http://127.0.0.1:8080/ckeditor5/35.4.0/installation/getting-started/getting-and-setting-data.html#demo 
## Expected:
image
## Actual:
image

Similar for: 
http://127.0.0.1:8080/ckeditor5/35.4.0/installation/getting-started/getting-and-setting-data.html#demo-2

## Additional info
Errors are present in the console:
```
Uncaught ReferenceError: ClassicEditor is not defined
    at eval (autosave.js:18:1)
    at ./node_modules/umberto/source/ckeditor5/35.4.0/_snippets/installation/getting-and-setting-data/autosave.js (snippet.js:23:1)
    at __webpack_require__ (snippet.js:57:41)
    at snippet.js:97:37
    at snippet.js:99:12
manualsave.js:19 Uncaught ReferenceError: ClassicEditor is not defined
    at eval (manualsave.js:19:1)
    at ./node_modules/umberto/source/ckeditor5/35.4.0/_snippets/installation/getting-and-setting-data/manualsave.js (snippet.js:23:1)
    at __webpack_require__ (snippet.js:57:41)
    at snippet.js:97:37
    at snippet.js:99:12
```

File: docs/installation/getting-started/getting-and-setting-data.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ---
-category: advanced
-order: 20
+category: getting-started
+order: 70
 ---
 
-{@snippet installation/saving-data/build-autosave-source}
+# Getting and setting data
```
Comment (dufipl) [inline docs/installation/getting-started/getting-and-setting-data.md:6]: Build import was missing here. It should look like that:
```suggestion
{@snippet installation/getting-and-setting-data/build-autosave-source}

# Getting and setting data
```

Comment (dufipl) [reply docs/installation/getting-started/getting-and-setting-data.md:8]: Fixed in https://github.com/ckeditor/ckeditor5/pull/13170/commits/d015ea071852c953517943f1f1d886baa027296b.

File: docs/installation/getting-started/predefined-builds.md [hunk 2]
```diff
@@ -92,10 +92,10 @@ After downloading and installing a predefined CKEditor 5 build in your applicati
 <script src="[ckeditor-build-path]/ckeditor.js"></script>
 ```
 
-Once the CKEditor script is loaded, you can {@link installation/getting-started/basic-api use the API} to create editors in your page.
+Once the CKEditor script is loaded, you can {@link installation/getting-started/editor-lifecycle use the API} to create editors in your page.
 
 <info-box>
-	The `build/ckeditor.js` file is generated in the [UMD format](https://github.com/umdjs/umd) so you can also import it into your application if you use CommonJS modules (like in Node.js) or AMD modules (like in Require.js). Read more in the {@link installation/getting-started/basic-api#umd-support Basic API guide}.
+	The `build/ckeditor.js` file is generated in the [UMD format](https://github.com/umdjs/umd) so you can also import it into your application if you use CommonJS modules (like in Node.js) or AMD modules (like in Require.js). Read more in the {@link installation/getting-started/predefined-builds#umd-support Basic API guide}.
```
Comment (dufipl) [inline docs/installation/getting-started/predefined-builds.md:98]: We refer to the Basic API guide that doesn't exist anymore. We should rephrase that.

```suggestion
	The `build/ckeditor.js` file is generated in the [UMD format](https://github.com/umdjs/umd) so you can also import it into your application if you use CommonJS modules (like in Node.js) or AMD modules (like in Require.js). Read more in the {@link installation/getting-started/predefined-builds#umd-support UMD support section guide}.
```

File: docs/updating/migration-from-ckeditor-4.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ There is no "drop in" solution for migrating. In this guide we hope to summarize
 
 The very first aspect that changed with CKEditor 5 is its installation procedure. It became much more modern with the introduction of modular patterns, UMD, npm, etc. Refer to {@link installation/getting-started/quick-start Installation} for more details.
 
-The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/basic-api Basic API} for an introduction.
+The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/editor-lifecycle Editor Lifecycle} and @link installation/getting-started/save-editor Getting and setting data} for an introduction.
```
Comment (dufipl) [inline docs/updating/migration-from-ckeditor-4.md:26]: This link is wrong.
```suggestion
The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/editor-lifecycle Editor Lifecycle} and @link installation/getting-started/getting-and-setting-data Getting and setting data} for an introduction.
```

File: docs/installation/migration-from-ckeditor-4.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ When compared to its predecessor, CKEditor 5 should be considered **a totally ne
 
 The very first aspect that changed with CKEditor 5 is its installation procedure. It became much more modern with the introduction of modular patterns, UMD, npm, etc. Refer to {@link installation/getting-started/quick-start-other Installation} for more details.
 
-The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/basic-api Basic API} for an introduction.
+The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/editor-lifecycle Editor Lifecycle} and @link installation/getting-started/save-editor Getting and setting data} for an introduction.
```
Comment (dufipl) [inline docs/installation/migration-from-ckeditor-4.md:26]: This link is wrong.
```suggestion
The API for integrating CKEditor with your pages also changed. It is worth checking {@link installation/getting-started/editor-lifecycle Editor Lifecycle} and @link installation/getting-started/getting-and-setting-data Getting and setting data} for an introduction.
```

**Review by dufipl — CHANGES_REQUESTED** [review]

File: docs/updating/migration-from-ckeditor-4.md (hunk 3 above)
Comment (dufipl) [reply docs/updating/migration-from-ckeditor-4.md:26]: @gorzelinski we still miss this one ☝️

**Review by dufipl — APPROVED** [review]
LGTM 🎉

**Pull request merged**